conda/plugins/subcommands/doctor/health_checks.py

```diff
@@ -5,22 +5,25 @@
 from __future__ import annotations
 
 import json
+import os
 from logging import getLogger
 from pathlib import Path
-from typing import TYPE_CHECKING
+
+import requests
```

Comment:
Please use the conda session for this (as it pre-configures authentication, TLS certs and proxy handling):

(please move this to the correct import line below)

```suggestion
from ...gateways.connection.session import get_session
```

See my suggestion below how to get the session object (which has the same interface as if you'd use the "requests" module you imported here first.